superset-frontend/src/explore/components/controls/DndColumnSelectControl/OptionWrapper.tsx

```diff
@@ -28,8 +28,21 @@ import {
   OptionProps,
   OptionItemInterface,
 } from 'src/explore/components/controls/DndColumnSelectControl/types';
+import { Tooltip } from 'src/components/Tooltip';
+import { StyledColumnOption } from 'src/explore/components/optionRenderers';
+import { styled } from '@superset-ui/core';
+import { ColumnMeta } from '@superset-ui/chart-controls';
 import Option from './Option';
 
+export const OptionLabel = styled.div`
+  width: 100%;
+  overflow: hidden;
+  text-overflow: ellipsis;
+  white-space: nowrap;
+`;
+
+const LabelText = styled.span``;
```

Comment:
Did you forget to add the props? If not, then it's better to remove this and use `span` directly.

Reply:
I think this was meant to get some custom style and then it didn't require it. I'll remove it.